.gitmodules

```diff
@@ -1,3 +1,6 @@
 [submodule "lib/dss-test"]
 	path = lib/dss-test
 	url = https://github.com/makerdao/dss-test
+[submodule "lib/endgame-toolkit"]
+	path = lib/endgame-toolkit
+	url = https://github.com/dewiz-xyz/endgame-toolkit
```

Comment:
I'd suggest avoiding this dependency and maybe have a mock or just the farm copied as an util or so.

Reply:
Changed

Reply:
Are we sure about this?
The nice think about having the dependency is that if we make changes in the endgame-toolkit code (which we are now discussing) we will have them here just by updating the dependency.
Also, there are some small changes in the mock that it might be nicer not to test with.

Reply:
I don't really like having an external dependency here. Makes the repos heavier and in future we might start having incompatibility issues due to solidity versions. If we do not want to test with a mock we might prefer to copy the real thing.

Reply:
Yeah still not sure we agree here, but not a hill I would die on.
Resolving.